presto-main/src/main/java/com/facebook/presto/execution/Input.java

```diff
@@ -37,6 +37,7 @@
     private final List<Column> columns;
     private final Optional<Object> connectorInfo;
     private final Optional<TableStatistics> statistics;
+    private final String serializedCommitOutput;
```

Comment:
This field is not intuitive for the `Input` class. Can we elaborate it a bit more?